src/urllib3/contrib/_securetransport/bindings.py

```diff
@@ -47,26 +46,27 @@
 from ctypes import CDLL, POINTER, CFUNCTYPE
 
 
-security_path = find_library("Security")
```

Comment:
I'm concerned about changing this code path for macOS 10.15 or lower. How does something like this look?

```python
from urllib3.packages.six import raise_from
version_info = tuple(map(int, version.split(".")))

def load_cdll(name, macos10_16_path):
    """Loads a CDLL with a fallback path for macOS Big Sur
    10.16/11+ due to the dynamic linker cache.
    """
    try:
        # Big Sur is technically 11 but we use 10.16 due
        # to the Big Sur beta being labeled as 10.16.
        if version_info >= (10, 16):
            path = macos10_16_path
        else:
            path = find_library(name)
        if not path:
            raise OSError  # Caught and reraised as 'ImportError'
        return CDLL(path, use_errno=True)
    except OSError:
        raise_from(ImportError("The library %s failed to load" % name), None)

Security = load_cdll("Security", "<path for Security>")
CoreFoundation = load_cdll("CoreFoundation", "<path for CF>")
```

Reply:
That works too. I can rework to do it that way. It's certainly safer. I love trimming and cleaning code out, but even though the paths _should_ be correct all the way back, I think it can't hurt to err on the safe side.

Reply:
Do you want to do it the other way around? Try `find_library` and if that comes up with `None`, use the second argument to `load_cdll`?

Kind of like `dict.get`

Reply:
That works for me too!